lang/clisp/Portfile

```diff
@@ -24,13 +26,12 @@ long_description        \
   Maxima, ACL2 and many other Common Lisp packages.
 
 homepage        https://www.clisp.org
-master_sites    https://alpha.gnu.org/gnu/clisp/
 
-checksums       md5     edb0e61a66693bfb246d157b75df195a \
-                sha1    7782db3789b12ac3a2f518b47bfbc13088c3ebb8 \
-                rmd160  7dea9e01b52b95b765fd0ea32cf613ecc5ca5b7a \
-                sha256  bd443a94aa9b02da4c4abbcecfc04ffff1919c0a8b0e7e35649b86198cd6bb89 \
-                size    8919616
+checksums       md5     b6ba91169bbacdd1084e1a3b8fb01232 \
+                sha1    0efb319872c5f1749eb44d4b6f32df2ddc73135c \
```

Comment:
`md5` and `sha1` checksums are obsolete and should be removed

Reply:
I have removed the checksums